src/components/swapv2/LimitOrder/LimitOrderForm.tsx

```diff
@@ -281,7 +281,9 @@ const LimitOrderForm = function LimitOrderForm({
 
   const handleMaxInput = useCallback(() => {
     if (!parsedActiveOrderMakingAmount || !maxAmountInput) return
-    onSetInput(maxAmountInput.subtract(parsedActiveOrderMakingAmount)?.toExact())
+    try {
+      onSetInput(maxAmountInput.subtract(parsedActiveOrderMakingAmount)?.toExact())
```

Comment:
Should we wrap `maxAmountInput` with `useCallback` to prevent it from calculating every time?